modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -179,13 +187,37 @@ export default {
       }
     }
   },
+  watch: {
+    'docUtilityFields.data': {
+      deep: true,
+      handler(newVal, oldVal) {
+        if (this.moduleName !== '@apostrophecms/page' || this.splittingDoc) {
+          return;
+        }
+
+        if (this.docType !== newVal.type) {
+          // Return the split data into `doc.data` before splitting again.
+          this.doc.data = defaultsDeep(this.docOtherFields.data, this.docUtilityFields.data, this.doc.data);
```

Comment:
The new schema may have a field with the same in name in a different tab, so get it all back together before splitting up again.

Reply:
I do think we should revisit this splitting pattern in beta phase. It doesn't feel great. CC @localghost443